packages/babel-plugin-bugfix-v8-spread-parameters-in-optional-chaining/src/index.ts

```diff
@@ -12,7 +14,9 @@ export default declare(api => {
     name: "bugfix-v8-spread-parameters-in-optional-chaining",
 
     visitor: {
-      "OptionalCallExpression|OptionalMemberExpression"(path) {
+      "OptionalCallExpression|OptionalMemberExpression"(
+        path: NodePath<t.OptionalCallExpression | t.OptionalMemberExpression>,
```

Comment:
We still have to manually annotate the input of the multiple-type selection visitors.